install/kubernetes/cilium/charts/hubble-ui/values.yaml

```diff
@@ -0,0 +1,99 @@
+# image used for hubble server
```

Comment:
All values _not_ part of `ui:` need to be removed from here, they should be port of the `hubble` section in the global Values:
https://github.com/cilium/cilium/blob/99a5aae2909f796d0e10341b9a2256444856eed4/install/kubernetes/cilium/values.yaml#L390-L400

Reply:
Will fix it in the next push. I might have missed it.

Reply:
I have moved most couple of items and removed items that are not required.
Not sure if we need the 'image' section here.
I think the 'ui' section and 'ingress' section can stay here.

Reply:
The `image` section should be removed as well. As of yesterday, `quay.io/cilium/hubble:latest` does not contain a standalone server anymore and therefore will not run at all. All it's functionality is now part of the `cilium/cilium` Docker image.

Reply:
Addressed review comments.